### PR TITLE
Handle creating toggles

### PR DESCRIPTION
**Pull request by rkondratowicz**
This PR adds config and code for creating toggles in the api.  
Test can also be run, but it only creates toggles (for now).

The plan is to create toggles in setup stage, then stress the api with reads and finally remove the toggles.

### CODE REVIEW

File: src/gatling/resources/conf/application.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+baseUrl = "http://localhost:8580"
```
Comment (rkondratowicz) [inline src/gatling/resources/conf/application.conf:1]: configured to run on local instance by default

File: src/gatling/scala/uk/gov/hmcts/reform/featuretoggle/MainSimulation.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  setUp(
+    scenario("Main scenario")
+      .during(1.minutes)(
+        exec(
+          createToggle
+          pause(1.seconds, 2.seconds)
+        )
+      )
+      .inject(
+        rampUsers(100).over(10.seconds)
```
Comment (rkondratowicz) [inline src/gatling/scala/uk/gov/hmcts/reform/featuretoggle/MainSimulation.scala:23]: duration time, number of users/clients etc are likely to change in future PRs.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # feature-toggle-performance-tests
 
+## Running
+```bash
+$ ./gradlew gatlingRun-uk.gov.hmcts.reform.featuretoggle.MainSimulation
```
Comment (doncem) [inline README.md:5]: huh, wouldn't guessed the syntax: defining which test to run but without spaces

**Review by doncem — APPROVED** [review]
yea you probably right, actual performance test would be if some features would be set up prior testing suite

**Pull request merged**